Controller/Blog/View.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mooore\WordpressIntegrationCms\Controller\Blog;
+
+use Magento\Customer\Model\Session;
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\App\Action\Context;
+use Mooore\WordpressIntegrationCms\Model\RemotePostRepository;
+use Magento\Framework\View\Result\Page as ResultPage;
+use \Magento\Framework\Controller\Result\ForwardFactory;
+use Magento\Framework\Controller\ResultFactory;
+
+class View extends Action implements HttpGetActionInterface
+{
+    /**
+     * @var RemotePostRepository
+     */
+    protected $remotePostRepository;
+
+    /**
+     * @var ResultPage
+     */
+    protected $resultPage;
+
+    /**
+     * @var ForwardFactory
+     */
+    protected $forwardFactory;
+
+    /**
+     * @var ResultFactory
+     */
+    protected $resultFactory;
+
+    public function __construct(
+        Context $context,
+        RemotePostRepository $remotePostRepository,
+        ResultPage $resultPage,
+        ForwardFactory $forwardFactory,
+        ResultFactory $resultFactory
+    ) {
+        $this->remotePostRepository = $remotePostRepository;
+        $this->resultPage = $resultPage;
+        $this->forwardFactory = $forwardFactory;
+        $this->resultFactory = $resultFactory;
+        parent::__construct($context);
+    }
+
+    public function execute()
+    {
+        // https://aic.aic.test/wpci/blog/view/site_id/2/page_id/1
+        $siteId = (int)$this->getRequest()->getParam('site_id', false);
+        $pageId = (int)$this->getRequest()->getParam('page_id', false);
+
+        $page = null;
+        try {
+            $page = $this->remotePostRepository->get($siteId, $pageId);
+        }
+        catch (\Exception $e) {
+            $page = null;
```

Comment:
Is it possible to add some kind of logging to this bit? That might make debugging a bit easier. :)

Reply:
Logging is already added in the parent function, so maybe logging would be unneeded here right now.